language/documentation/examples/diem-framework/crates/crypto/src/ed25519.rs

```diff
@@ -161,7 +161,7 @@ impl Ed25519PublicKey {
             bits
         };
         let mtg_point = curve25519_dalek::montgomery::MontgomeryPoint(key_bits);
-        let sign = if negative { 1u8 } else { 0u8 };
+        let sign = u8::from(negative);
```

Comment:
This isn't new right? Just a clippy thing?

Reply:
Just a clippy thing, yeah